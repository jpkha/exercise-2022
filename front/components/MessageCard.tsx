import {Message} from '../model/message';

interface MessageProps {
  readonly message: Message,
  readonly handleOnClickMessage: (id: string) => {}
}

export const MessageCard = ({message, handleOnClickMessage}: MessageProps) => {
  return <div onClick={() => handleOnClickMessage(message.id)}>
    <p>
      {message.subject}
    </p>
    <p>
      {message.body}
    </p>
  </div>
}
