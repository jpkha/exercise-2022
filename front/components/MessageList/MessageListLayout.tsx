import {MessagesListContainer} from './MessageListContainer';
import {Message} from '../../model/api/message';

const MessageListLayout = ({messagesData}: { messagesData: Message[] }) => {
  return (<MessagesListContainer messagesData={messagesData}/>)
};

export default MessageListLayout;
