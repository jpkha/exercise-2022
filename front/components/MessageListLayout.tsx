import {MessagesListContainer} from './MessageListContainer';
import {useRouter} from 'next/router';
import {Message} from '../model/message';

const MessageListLayout = ({messagesData}: { messagesData: Message[] }) => {
  const router = useRouter();
  const realtorsId = router.query.realtorsId as string;

  const handleOnClickMessage = (message: string) => {
    router.push(`/realtors/${realtorsId}/messages/${message}`, undefined, {shallow: true});
  }
  return (<MessagesListContainer messagesData={messagesData} handleOnClickMessage={handleOnClickMessage}/>)
};

export default MessageListLayout;
