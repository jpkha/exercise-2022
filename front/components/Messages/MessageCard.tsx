import {MessageProps} from '../../model/messageProps';
import {MessageEmailCard} from './MessageEmailCard';
import {MessageSmsCard} from './MessageSmsCard';
import {MessagePhoneCard} from './MessagePhoneCard';

export const MessageCard = ({message}: MessageProps) => {
  switch (message.type) {
    case 'email':
      return <MessageEmailCard message={message}/>
    case 'sms':
      return <MessageSmsCard message={message}/>
    case 'phone':
      return <MessagePhoneCard message={message}/>
    default:
      return <div>Error, need to be checked</div>
  }
}
