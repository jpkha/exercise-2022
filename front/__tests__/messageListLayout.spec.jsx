// __tests__/headerApp.spec.jsx

/**
 * @jest-environment jsdom
 */

import React from 'react'
import {render, screen} from '@testing-library/react'
import * as nextRouter from 'next/router';
import MessageListLayout from '../components/MessageList/MessageListLayout';
import {mockMessagesPage} from '../__mocks__/mockMessagesPage';
import {MessageCard} from '../components/Messages/MessageCard';
import {mockMessageEmail, mockMessagePhone, mockMessageSms} from '../__mocks__/mockMessage';

describe('MessageListLayout', () => {
    const push = jest.fn();
    beforeEach(() => {
        nextRouter.useRouter = jest.fn();
        nextRouter.useRouter.mockImplementation(() => ({
            query: {realtorsId: '101'},
            push
        }));
    })

    it('should render correctly a list of message', () => {
        const tree = render(<>
            <MessageListLayout messagesData={mockMessagesPage}/>
        </>)
        expect(tree).toMatchSnapshot();
    })
    it('should render a list of message', () => {
        render(
            <MessageListLayout messagesData={mockMessagesPage}/>
        )
        const messageList = screen.getByTestId('message-list-container');
        expect(messageList).toBeInTheDocument()
        const listItem = screen.getAllByRole('listitem');
        expect(listItem.length).toBe(10);
        expect(listItem[0]).toBeInTheDocument();
    })

})
