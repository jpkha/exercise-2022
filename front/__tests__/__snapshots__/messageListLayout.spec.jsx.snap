// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageListLayout should render correctly a list of message 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <ul
        class="sc-iCfMLu bUJXye"
        data-testid="message-list-container"
        id="message-list-container"
      >
        <li
          aria-label="Message lu de Charles Davis"
          class="sc-dkPtRN fCCaPm"
          data-testid-read="true"
        >
          <div
            aria-label="Icône Message lu"
            class="sc-gKclnd gciYjb"
            role="img"
          >
            <i
              class="mypro-icon mypro-icon-mail"
            />
          </div>
          <div
            class="sc-hKwDye fXlfxa"
          >
            <div
              class="sc-bdvvtL bmAa-dP"
            >
              <h2>
                <a
                  class="sc-gsDKAQ kedSOv"
                  href="/realtors/101/messages/10100"
                >
                  Charles Davis
                </a>
              </h2>
              <time>
                30/12/2021
              </time>
            </div>
            <p
              class="sc-eCImPb bgoaAA"
            >
              Message sur votre vitrine Meilleurs Agents
              <small
                class="sc-jRQBWg bxKdkc"
              >
                Lorem Ipsum #10100 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
              </small>
            </p>
          </div>
        </li>
        <li
          aria-label="Message vocal non lu de William Johnson"
          class="sc-dkPtRN fCCaPm"
          data-testid-read="false"
        >
          <div
            aria-label="Icône Message vocal non lu"
            class="sc-gKclnd gTWMTY"
            role="img"
          >
            <i
              class="mypro-icon mypro-icon-phone"
            />
          </div>
          <div
            class="sc-hKwDye hzyTlS"
          >
            <div
              class="sc-bdvvtL dTcqam"
            >
              <h2>
                <a
                  class="sc-gsDKAQ kedSOv"
                  href="/realtors/101/messages/10101"
                >
                  William Johnson
                </a>
                <span>
                  (
                  06 84 51 52 33
                  )
                </span>
              </h2>
              <time>
                30/12/2021
              </time>
            </div>
            <p
              class="sc-eCImPb bgoaAA"
            >
              Message vocal sur votre vitrine Meilleurs Agents
              <small
                class="sc-jRQBWg bxKdkc"
              >
                Lorem Ipsum #10101 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
              </small>
            </p>
          </div>
        </li>
        <li
          aria-label="Message vocal non lu de John Williams"
          class="sc-dkPtRN fCCaPm"
          data-testid-read="false"
        >
          <div
            aria-label="Icône Message vocal non lu"
            class="sc-gKclnd gTWMTY"
            role="img"
          >
            <i
              class="mypro-icon mypro-icon-phone"
            />
          </div>
          <div
            class="sc-hKwDye hzyTlS"
          >
            <div
              class="sc-bdvvtL dTcqam"
            >
              <h2>
                <a
                  class="sc-gsDKAQ kedSOv"
                  href="/realtors/101/messages/10102"
                >
                  John Williams
                </a>
                <span>
                  (
                  06 37 49 84 92
                  )
                </span>
              </h2>
              <time>
                05/01/2022
              </time>
            </div>
            <p
              class="sc-eCImPb bgoaAA"
            >
              Message vocal sur votre vitrine Meilleurs Agents
              <small
                class="sc-jRQBWg bxKdkc"
              >
                Lorem Ipsum #10102 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
              </small>
            </p>
          </div>
        </li>
        <li
          aria-label="Message non lu de James Jones"
          class="sc-dkPtRN fCCaPm"
          data-testid-read="false"
        >
          <div
            aria-label="Icône Message non lu"
            class="sc-gKclnd gTWMTY"
            role="img"
          >
            <i
              class="mypro-icon mypro-icon-inbox"
            />
          </div>
          <div
            class="sc-hKwDye hzyTlS"
          >
            <div
              class="sc-bdvvtL dTcqam"
            >
              <h2>
                <a
                  class="sc-gsDKAQ kedSOv"
                  href="/realtors/101/messages/10103"
                >
                  James Jones
                </a>
              </h2>
              <time>
                24/12/2021
              </time>
            </div>
            <p
              class="sc-eCImPb bgoaAA"
            >
              Message sur votre vitrine Meilleurs Agents
              <small
                class="sc-jRQBWg bxKdkc"
              >
                Lorem Ipsum #10103 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
              </small>
            </p>
          </div>
        </li>
        <li
          aria-label="Message vocal non lu de Michael Davis"
          class="sc-dkPtRN fCCaPm"
          data-testid-read="false"
        >
          <div
            aria-label="Icône Message vocal non lu"
            class="sc-gKclnd gTWMTY"
            role="img"
          >
            <i
              class="mypro-icon mypro-icon-phone"
            />
          </div>
          <div
            class="sc-hKwDye hzyTlS"
          >
            <div
              class="sc-bdvvtL dTcqam"
            >
              <h2>
                <a
                  class="sc-gsDKAQ kedSOv"
                  href="/realtors/101/messages/10104"
                >
                  Michael Davis
                </a>
                <span>
                  (
                  06 81 73 18 80
                  )
                </span>
              </h2>
              <time>
                22/12/2021
              </time>
            </div>
            <p
              class="sc-eCImPb bgoaAA"
            >
              Message vocal sur votre vitrine Meilleurs Agents
              <small
                class="sc-jRQBWg bxKdkc"
              >
                Lorem Ipsum #10104 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
              </small>
            </p>
          </div>
        </li>
        <li
          aria-label="Message non lu de John Taylor"
          class="sc-dkPtRN fCCaPm"
          data-testid-read="false"
        >
          <div
            aria-label="Icône Message non lu"
            class="sc-gKclnd gTWMTY"
            role="img"
          >
            <i
              class="mypro-icon mypro-icon-inbox"
            />
          </div>
          <div
            class="sc-hKwDye hzyTlS"
          >
            <div
              class="sc-bdvvtL dTcqam"
            >
              <h2>
                <a
                  class="sc-gsDKAQ kedSOv"
                  href="/realtors/101/messages/10105"
                >
                  John Taylor
                </a>
              </h2>
              <time>
                30/12/2021
              </time>
            </div>
            <p
              class="sc-eCImPb bgoaAA"
            >
              Message sur votre vitrine Meilleurs Agents
              <small
                class="sc-jRQBWg bxKdkc"
              >
                Lorem Ipsum #10105 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
              </small>
            </p>
          </div>
        </li>
        <li
          aria-label="Message lu de David Jackson"
          class="sc-dkPtRN fCCaPm"
          data-testid-read="true"
        >
          <div
            aria-label="Icône Message lu"
            class="sc-gKclnd gciYjb"
            role="img"
          >
            <i
              class="mypro-icon mypro-icon-mail"
            />
          </div>
          <div
            class="sc-hKwDye fXlfxa"
          >
            <div
              class="sc-bdvvtL bmAa-dP"
            >
              <h2>
                <a
                  class="sc-gsDKAQ kedSOv"
                  href="/realtors/101/messages/10106"
                >
                  David Jackson
                </a>
              </h2>
              <time>
                08/01/2022
              </time>
            </div>
            <p
              class="sc-eCImPb bgoaAA"
            >
              Message sur votre vitrine Meilleurs Agents
              <small
                class="sc-jRQBWg bxKdkc"
              >
                Lorem Ipsum #10106 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
              </small>
            </p>
          </div>
        </li>
        <li
          aria-label="Message vocal non lu de William Smith"
          class="sc-dkPtRN fCCaPm"
          data-testid-read="false"
        >
          <div
            aria-label="Icône Message vocal non lu"
            class="sc-gKclnd gTWMTY"
            role="img"
          >
            <i
              class="mypro-icon mypro-icon-phone"
            />
          </div>
          <div
            class="sc-hKwDye hzyTlS"
          >
            <div
              class="sc-bdvvtL dTcqam"
            >
              <h2>
                <a
                  class="sc-gsDKAQ kedSOv"
                  href="/realtors/101/messages/10107"
                >
                  William Smith
                </a>
                <span>
                  (
                  06 90 62 34 89
                  )
                </span>
              </h2>
              <time>
                26/12/2021
              </time>
            </div>
            <p
              class="sc-eCImPb bgoaAA"
            >
              Message vocal sur votre vitrine Meilleurs Agents
              <small
                class="sc-jRQBWg bxKdkc"
              >
                Lorem Ipsum #10107 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
              </small>
            </p>
          </div>
        </li>
        <li
          aria-label="SMS non lu de Richard Davis"
          class="sc-dkPtRN fCCaPm"
          data-testid-read="false"
        >
          <div
            aria-label="Icône SMS non lu"
            class="sc-gKclnd gTWMTY"
            role="img"
          >
            <i
              class="mypro-icon mypro-icon-sms"
            />
          </div>
          <div
            class="sc-hKwDye hzyTlS"
          >
            <div
              class="sc-bdvvtL dTcqam"
            >
              <h2>
                <a
                  class="sc-gsDKAQ kedSOv"
                  href="/realtors/101/messages/10108"
                >
                  Richard Davis
                </a>
                <span>
                  (
                  06 85 82 34 30
                  )
                </span>
              </h2>
              <time>
                19/12/2021
              </time>
            </div>
            <p
              class="sc-eCImPb bgoaAA"
            >
              SMS sur votre vitrine Meilleurs Agents
              <small
                class="sc-jRQBWg bxKdkc"
              >
                Lorem Ipsum #10108 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
              </small>
            </p>
          </div>
        </li>
        <li
          aria-label="Message lu de William Williams"
          class="sc-dkPtRN fCCaPm"
          data-testid-read="true"
        >
          <div
            aria-label="Icône Message lu"
            class="sc-gKclnd gciYjb"
            role="img"
          >
            <i
              class="mypro-icon mypro-icon-mail"
            />
          </div>
          <div
            class="sc-hKwDye fXlfxa"
          >
            <div
              class="sc-bdvvtL bmAa-dP"
            >
              <h2>
                <a
                  class="sc-gsDKAQ kedSOv"
                  href="/realtors/101/messages/10109"
                >
                  William Williams
                </a>
              </h2>
              <time>
                04/01/2022
              </time>
            </div>
            <p
              class="sc-eCImPb bgoaAA"
            >
              Message sur votre vitrine Meilleurs Agents
              <small
                class="sc-jRQBWg bxKdkc"
              >
                Lorem Ipsum #10109 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
              </small>
            </p>
          </div>
        </li>
      </ul>
    </div>
  </body>,
  "container": <div>
    <ul
      class="sc-iCfMLu bUJXye"
      data-testid="message-list-container"
      id="message-list-container"
    >
      <li
        aria-label="Message lu de Charles Davis"
        class="sc-dkPtRN fCCaPm"
        data-testid-read="true"
      >
        <div
          aria-label="Icône Message lu"
          class="sc-gKclnd gciYjb"
          role="img"
        >
          <i
            class="mypro-icon mypro-icon-mail"
          />
        </div>
        <div
          class="sc-hKwDye fXlfxa"
        >
          <div
            class="sc-bdvvtL bmAa-dP"
          >
            <h2>
              <a
                class="sc-gsDKAQ kedSOv"
                href="/realtors/101/messages/10100"
              >
                Charles Davis
              </a>
            </h2>
            <time>
              30/12/2021
            </time>
          </div>
          <p
            class="sc-eCImPb bgoaAA"
          >
            Message sur votre vitrine Meilleurs Agents
            <small
              class="sc-jRQBWg bxKdkc"
            >
              Lorem Ipsum #10100 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
            </small>
          </p>
        </div>
      </li>
      <li
        aria-label="Message vocal non lu de William Johnson"
        class="sc-dkPtRN fCCaPm"
        data-testid-read="false"
      >
        <div
          aria-label="Icône Message vocal non lu"
          class="sc-gKclnd gTWMTY"
          role="img"
        >
          <i
            class="mypro-icon mypro-icon-phone"
          />
        </div>
        <div
          class="sc-hKwDye hzyTlS"
        >
          <div
            class="sc-bdvvtL dTcqam"
          >
            <h2>
              <a
                class="sc-gsDKAQ kedSOv"
                href="/realtors/101/messages/10101"
              >
                William Johnson
              </a>
              <span>
                (
                06 84 51 52 33
                )
              </span>
            </h2>
            <time>
              30/12/2021
            </time>
          </div>
          <p
            class="sc-eCImPb bgoaAA"
          >
            Message vocal sur votre vitrine Meilleurs Agents
            <small
              class="sc-jRQBWg bxKdkc"
            >
              Lorem Ipsum #10101 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
            </small>
          </p>
        </div>
      </li>
      <li
        aria-label="Message vocal non lu de John Williams"
        class="sc-dkPtRN fCCaPm"
        data-testid-read="false"
      >
        <div
          aria-label="Icône Message vocal non lu"
          class="sc-gKclnd gTWMTY"
          role="img"
        >
          <i
            class="mypro-icon mypro-icon-phone"
          />
        </div>
        <div
          class="sc-hKwDye hzyTlS"
        >
          <div
            class="sc-bdvvtL dTcqam"
          >
            <h2>
              <a
                class="sc-gsDKAQ kedSOv"
                href="/realtors/101/messages/10102"
              >
                John Williams
              </a>
              <span>
                (
                06 37 49 84 92
                )
              </span>
            </h2>
            <time>
              05/01/2022
            </time>
          </div>
          <p
            class="sc-eCImPb bgoaAA"
          >
            Message vocal sur votre vitrine Meilleurs Agents
            <small
              class="sc-jRQBWg bxKdkc"
            >
              Lorem Ipsum #10102 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
            </small>
          </p>
        </div>
      </li>
      <li
        aria-label="Message non lu de James Jones"
        class="sc-dkPtRN fCCaPm"
        data-testid-read="false"
      >
        <div
          aria-label="Icône Message non lu"
          class="sc-gKclnd gTWMTY"
          role="img"
        >
          <i
            class="mypro-icon mypro-icon-inbox"
          />
        </div>
        <div
          class="sc-hKwDye hzyTlS"
        >
          <div
            class="sc-bdvvtL dTcqam"
          >
            <h2>
              <a
                class="sc-gsDKAQ kedSOv"
                href="/realtors/101/messages/10103"
              >
                James Jones
              </a>
            </h2>
            <time>
              24/12/2021
            </time>
          </div>
          <p
            class="sc-eCImPb bgoaAA"
          >
            Message sur votre vitrine Meilleurs Agents
            <small
              class="sc-jRQBWg bxKdkc"
            >
              Lorem Ipsum #10103 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
            </small>
          </p>
        </div>
      </li>
      <li
        aria-label="Message vocal non lu de Michael Davis"
        class="sc-dkPtRN fCCaPm"
        data-testid-read="false"
      >
        <div
          aria-label="Icône Message vocal non lu"
          class="sc-gKclnd gTWMTY"
          role="img"
        >
          <i
            class="mypro-icon mypro-icon-phone"
          />
        </div>
        <div
          class="sc-hKwDye hzyTlS"
        >
          <div
            class="sc-bdvvtL dTcqam"
          >
            <h2>
              <a
                class="sc-gsDKAQ kedSOv"
                href="/realtors/101/messages/10104"
              >
                Michael Davis
              </a>
              <span>
                (
                06 81 73 18 80
                )
              </span>
            </h2>
            <time>
              22/12/2021
            </time>
          </div>
          <p
            class="sc-eCImPb bgoaAA"
          >
            Message vocal sur votre vitrine Meilleurs Agents
            <small
              class="sc-jRQBWg bxKdkc"
            >
              Lorem Ipsum #10104 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
            </small>
          </p>
        </div>
      </li>
      <li
        aria-label="Message non lu de John Taylor"
        class="sc-dkPtRN fCCaPm"
        data-testid-read="false"
      >
        <div
          aria-label="Icône Message non lu"
          class="sc-gKclnd gTWMTY"
          role="img"
        >
          <i
            class="mypro-icon mypro-icon-inbox"
          />
        </div>
        <div
          class="sc-hKwDye hzyTlS"
        >
          <div
            class="sc-bdvvtL dTcqam"
          >
            <h2>
              <a
                class="sc-gsDKAQ kedSOv"
                href="/realtors/101/messages/10105"
              >
                John Taylor
              </a>
            </h2>
            <time>
              30/12/2021
            </time>
          </div>
          <p
            class="sc-eCImPb bgoaAA"
          >
            Message sur votre vitrine Meilleurs Agents
            <small
              class="sc-jRQBWg bxKdkc"
            >
              Lorem Ipsum #10105 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
            </small>
          </p>
        </div>
      </li>
      <li
        aria-label="Message lu de David Jackson"
        class="sc-dkPtRN fCCaPm"
        data-testid-read="true"
      >
        <div
          aria-label="Icône Message lu"
          class="sc-gKclnd gciYjb"
          role="img"
        >
          <i
            class="mypro-icon mypro-icon-mail"
          />
        </div>
        <div
          class="sc-hKwDye fXlfxa"
        >
          <div
            class="sc-bdvvtL bmAa-dP"
          >
            <h2>
              <a
                class="sc-gsDKAQ kedSOv"
                href="/realtors/101/messages/10106"
              >
                David Jackson
              </a>
            </h2>
            <time>
              08/01/2022
            </time>
          </div>
          <p
            class="sc-eCImPb bgoaAA"
          >
            Message sur votre vitrine Meilleurs Agents
            <small
              class="sc-jRQBWg bxKdkc"
            >
              Lorem Ipsum #10106 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
            </small>
          </p>
        </div>
      </li>
      <li
        aria-label="Message vocal non lu de William Smith"
        class="sc-dkPtRN fCCaPm"
        data-testid-read="false"
      >
        <div
          aria-label="Icône Message vocal non lu"
          class="sc-gKclnd gTWMTY"
          role="img"
        >
          <i
            class="mypro-icon mypro-icon-phone"
          />
        </div>
        <div
          class="sc-hKwDye hzyTlS"
        >
          <div
            class="sc-bdvvtL dTcqam"
          >
            <h2>
              <a
                class="sc-gsDKAQ kedSOv"
                href="/realtors/101/messages/10107"
              >
                William Smith
              </a>
              <span>
                (
                06 90 62 34 89
                )
              </span>
            </h2>
            <time>
              26/12/2021
            </time>
          </div>
          <p
            class="sc-eCImPb bgoaAA"
          >
            Message vocal sur votre vitrine Meilleurs Agents
            <small
              class="sc-jRQBWg bxKdkc"
            >
              Lorem Ipsum #10107 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
            </small>
          </p>
        </div>
      </li>
      <li
        aria-label="SMS non lu de Richard Davis"
        class="sc-dkPtRN fCCaPm"
        data-testid-read="false"
      >
        <div
          aria-label="Icône SMS non lu"
          class="sc-gKclnd gTWMTY"
          role="img"
        >
          <i
            class="mypro-icon mypro-icon-sms"
          />
        </div>
        <div
          class="sc-hKwDye hzyTlS"
        >
          <div
            class="sc-bdvvtL dTcqam"
          >
            <h2>
              <a
                class="sc-gsDKAQ kedSOv"
                href="/realtors/101/messages/10108"
              >
                Richard Davis
              </a>
              <span>
                (
                06 85 82 34 30
                )
              </span>
            </h2>
            <time>
              19/12/2021
            </time>
          </div>
          <p
            class="sc-eCImPb bgoaAA"
          >
            SMS sur votre vitrine Meilleurs Agents
            <small
              class="sc-jRQBWg bxKdkc"
            >
              Lorem Ipsum #10108 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
            </small>
          </p>
        </div>
      </li>
      <li
        aria-label="Message lu de William Williams"
        class="sc-dkPtRN fCCaPm"
        data-testid-read="true"
      >
        <div
          aria-label="Icône Message lu"
          class="sc-gKclnd gciYjb"
          role="img"
        >
          <i
            class="mypro-icon mypro-icon-mail"
          />
        </div>
        <div
          class="sc-hKwDye fXlfxa"
        >
          <div
            class="sc-bdvvtL bmAa-dP"
          >
            <h2>
              <a
                class="sc-gsDKAQ kedSOv"
                href="/realtors/101/messages/10109"
              >
                William Williams
              </a>
            </h2>
            <time>
              04/01/2022
            </time>
          </div>
          <p
            class="sc-eCImPb bgoaAA"
          >
            Message sur votre vitrine Meilleurs Agents
            <small
              class="sc-jRQBWg bxKdkc"
            >
              Lorem Ipsum #10109 is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
            </small>
          </p>
        </div>
      </li>
    </ul>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
